### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by jmcrey** [jamessward/codecademy-python#6]
## Rubric Score
 
### Criteria 1: Valid Python Code
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ The code included runs without any errors.
 
### Criteria 2: Implementation of Project Requirements
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ The code produces the suite of functions and classes required of it, and calls each of them in an appropriate order. Also, I love the addition of the `__eq__` function in the `TomeRater` class! Really nice job!
 
### Criteria 3: Software Architecture
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ The code is separated into distinct classes and functions, each of which are invoked for their own purposes. 
 
### Criteria 4: Uses Python Language Features
* _Score Level:_ 4 (Exceeds Expectations... by a lot)
* _Comment(s):_ Okay, this is where this project shined -- if there was a task that could be solved with a Python language built-in feature, it was. Some of my favorites are the use of `update`, `get`, and `locals` functions in their respective places. Although, by far, the use of `locals` was most impressive -- that was an extremely clever way to capture *all* potential values! Amazing job using all these builtin functions!
 
### Overall Score: 16/16

Simply put, this project was excellent. All the functions were accurate, the code conformed to Python's best practices, and the code base greatly leveraged Python's builtin features and functions. Not only that, but it included extra functions (such as the `__eq__` function) which added some nice features to the code base. For the future, there were some places that the code felt a bit over-complicated and could have been _slightly_ optimized (#2 and #4), but, still, all the functions were very precise. 

Fantastic job with this project!
